packages/protocol/contracts/L1/verifiers/PseZkVerifier.sol

```diff
@@ -44,49 +44,42 @@ contract PseZkVerifier is EssentialContract, IVerifier {
 
     /// @inheritdoc IVerifier
     function verifyProof(
-        uint64, /*blockId*/
-        address prover,
-        bool isContesting,
-        bool usingBlob,
-        bytes32 blobHash,
+        VerifierInput calldata input,
         TaikoData.BlockEvidence calldata evidence
     )
         external
         view
     {
         // Do not run proof verification to contest an existing proof
-        if (isContesting) return;
+        if (input.isContesting) return;
 
         PseZkEvmProof memory proof = abi.decode(evidence.proof, (PseZkEvmProof));
 
         bytes32 instance;
-
-        if (usingBlob) {
+        if (input.blobUsed) {
             PointProof memory pf = abi.decode(proof.pointProof, (PointProof));
-            bytes32 x = keccak256(abi.encodePacked(blobHash, pf.txListHash));
+
+            instance = calcInstance({
+                prover: input.prover,
```

Comment:
yes, instance must have `metaHash` (no longer part of evidence),  `txListHash` to be hashed in.

Reply:
Suggest to use evidence as the first param

Reply:
> yes, instance must have `metaHash` (no longer part of evidence), `txListHash` to be hashed in.

Please see my thoughts on why not txListHash has to be hashed in.
https://github.com/taikoxyz/taiko-mono/pull/15031#discussion_r1372232411

Reply:
I think it is still needed, if not, how can the circuits check that the right value is used for calculating the x: 

```
 Lib4844.evaluatePoint({
                blobHash: input.blobHash,
                x: uint256(
                    keccak256(abi.encodePacked(input.blobHash, pf.txListHash))
                    ) % Lib4844.BLS_MODULUS,
                y: pf.pointValue,
                commitment: pf.pointCommitment,
                proof: pf.pointProof
            });
```